DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/Controllers/RepositoryController.cs

```diff
@@ -49,15 +51,16 @@ public RepositoryController(IGithubService guGithubService, ISecurityContext sec
             return Ok(repos);
         }
 
+        // GET
         [Authorize]
-        [Route("api/repos/{repoName}/collaborators")]
-        public async Task<IHttpActionResult> GetCollaboratorsForRepo(string repoName)
+        [Route(RouteConstant.getCollaboratorsForRepository)]
+        public async Task<IHttpActionResult> GetCollaboratorsForRepo(string repositoryName)
         {
             var claims = User.Identity as ClaimsIdentity;
             string token = claims.FindFirstValue("ExternalAccessToken");
```

Comment:
Please move all staff related to ClaimsIdentity to extension methods. I assumed that such type od code will be used in another parts of the system.